.travis/before_install.sh

```diff
@@ -45,6 +45,11 @@ cd /tmp || die
     sudo -H pip install pexpect || die
     pip install pexpect || die
 
+    # posix-ncp tests
+    pip install scapy || die
```

Comment:
NCP tests were separated from CLI tests.  This PR integrates some NCP tests with the CLI tests.  Let's keep the model consistent.